index.html

```diff
@@ -205,34 +205,37 @@ <h2>Frequently Asked Questions</h2>
     </section>
 
     <!-- Contact -->
+    <section id="contact">
+        <div id="contact-content">
+            <h2>Contact</h2>
+            <p>Feel free to let us know of any questions, comments, and/or concerns at <a href="mailto:hello@brickhack.io">hello@brickhack.io</a></p>
+            <p>Interested in joining our team? <a href="./club.html">Check out our club</a></p>
+        </div>
+        <div id="contact-socials">
+            <a href="https://twitter.com/brickhackrit" target="_blank">
+                <i class="fab fa-twitter"></i>
+            </a>
+            <a href="https://www.facebook.com/brickhackrit" target="_blank">
+                <i class="fab fa-facebook"></i>
+            </a>
+            <a href="https://www.instagram.com/brickhack/" target="_blank">
+                <i class="fab fa-instagram"></i>
+            </a>
+            <a href="https://snapchat.com/add/brickhack" target="_blank">
+                <i class="fab fa-snapchat"></i>
+            </a>
+        </div>
+    </section>
+
+    <!-- Footer -->
     <footer>
-        <div id="footer-content">
-            <div>
-                <h2>contact us</h2>
-                <p><a href="mailto:hello@brickhack.io">hello@brickhack.io</a></p>
-            </div>
-            <div>
-                <a href="https://twitter.com/coderit" target="_blank">
-                    <i class="fab fa-twitter"></i>
-                </a>
-                <a href="https://www.facebook.com/brickhackrit" target="_blank">
-                    <i class="fab fa-facebook"></i>
-                </a>
-                <a href="https://www.instagram.com/code.rit/" target="_blank">
-                    <i class="fab fa-instagram"></i>
-                </a>
-                <a href="https://github.com/codeRIT" target="_blank">
-                    <i class="fab fa-github"></i>
-                </a>
-            </div>
-            <div>
-                <p>
-                    <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">code of conduct</a>
-                    |
-                    <a href="https://www.rit.edu/academicaffairs/policiesmanual/c070" target="_blank">privacy policy</a>
-                </p>
-                <p>copyright &#169; 2021 BrickHack Club</p>
-            </div>
+        <div id="footer-left">
+            <p><a href="mailto:hello@brickhack.io">hello@brickhack.io</a></p>
+        </div>
+        <div id="footer-right">
+            <p><a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf" target="_blank">MLH Code of Conduct</a></p>
+            <p><a href="https://www.rit.edu/academicaffairs/policiesmanual/c070" target="_blank">Privacy Policy</a></p>
+            <p>Copyright &#169; 2021 BrickHack Club</p>
```

Comment:
Can we use entity names instead of numbers for better code readability?
```suggestion
            <p>Copyright &copy; 2021 BrickHack Club</p>
```